frontend/packages/process-editor/src/hooks/useBpmnEditor.ts

```diff
@@ -123,12 +122,20 @@ export const useBpmnEditor = (): UseBpmnViewerResult => {
     modelerRef.current = getModeler(canvasRef.current);
     initializeEditor();
     initializeBpmnChanges();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Great that you add this! 👏 
Should we add something similar, as well as a comment on the GithubActions complaining on line 138? in this file too? 😄